code/modules/mob/living/simple_animal/animals/fish_vr.dm

```diff
@@ -34,9 +37,24 @@
 /mob/living/simple_animal/fish/koi/poisonous/proc/sting(var/mob/living/M)
 	if(!M.reagents)
 		return 0
-	M.reagents.add_reagent("carpotoxin", 3)
+	M.reagents.add_reagent("koipotoxin", 2)
+	M.reagents.add_reagent("impedrezene", 1)
 	return 1
 
+/datum/reagent/toxin/koipotoxin
+	name = "Koipotoxin"
+	id = "koipotoxin"
+	description = "A deadly neurotoxin legally distinct from carpotoxin."
+	taste_description = "fish"
+	reagent_state = LIQUID
+	color = "#003333"
+	strength = 10
+
 /obj/item/weapon/reagent_containers/food/snacks/carpmeat/koi
-	name = "koi fillet"
-	desc = "A fillet of genetically modified koi meat."
\ No newline at end of file
+	name = "fillet"
+	desc = "A fillet of genetically modified koi meat."
+
+/obj/item/weapon/reagent_containers/food/snacks/carpmeat/koi/New()
```

Comment:
alright, they do inherit reagents but also generate new ones, I'll axe the koi fillets and replace the snowflake poison with something else